.travis.yml

```diff
@@ -29,7 +29,7 @@ notifications:
 env:
   global:
     # Reduce the memory pressure on Travis CI to reduce build failures.
-    - GRADLE_OPTS="-Xms128m"
+    - GRADLE_OPTS="-Xms256m"
```

Comment:
Why do we need to increase this ?

Reply:
Our travis build is flaky due to failures from testFlowManagerListenerCreateContainerError. This fixes that problem.

Reply:
Good to know, thanks for taking care of this.